### ISSUE TITLE
usage double spaces text

### ISSUE THREAD

**Issue by karai17** [lunarmodules/ldoc#212]
![image](https://cloud.githubusercontent.com/assets/590332/9918359/bbb800a6-5c9c-11e5-9be8-faedc6cfa9e3.png)

As you can see, the `usage` tag is adding extra lines to my code block (actual extra lines, not just double-spaced lines).

``` lua
-- @usage
--  -- Create a Custom Layer
--  map:addCustomLayer("Sprite Layer", 3)
--
--  -- Add data to Custom Layer
--  local spriteLayer = map.layers["Sprite Layer"]
--  spriteLayer.sprites = {
--      player = {
--          image = love.graphics.newImage("assets/sprites/player.png"),
--          x = 64,
--          y = 64,
--          r = 0,
--      }
--  }
```

I would also like to note that issue #191 is relevant here.

**Comment by stevedonovan** [comment]
Interesting! Can't reproduce, will test later on Windows. Could be a \r\n weirdness.

**Comment by karai17** [comment]
Yeah, this was rendered on a Windows machine. I'm now running Linux and my most recent output displays normally except for the error noted in issue #191:

![image](https://cloud.githubusercontent.com/assets/590332/16381609/b90c6ad0-3c53-11e6-98c7-a1714991f0c6.png)

**Comment by stevedonovan** [comment]
Ah, thought so! Then I know what the problem is. Let me look at #191

**Comment by alerque** [comment]
Fixed in 7bed1fa.
